front/pages/api/w/[wId]/data_sources/[name]/managed/content-nodes/from-tables.ts

```diff
@@ -0,0 +1,187 @@
+import type { ContentNode, WithAPIErrorReponse } from "@dust-tt/types";
```

Comment:
Why do we need that endpoint? I don't see it used? Aren't we supposed to get away with the default content node flow?

Reply:
By that I mean, from the tableId we use a types function that does the mapping tableId => contentNodeId and use the default content node flow

Reply:
We don’t currently have a way to pull the nodes from the client (we use serverside props). But now that you say it, API should be internal IDs not table IDs

Reply:
\o/